doc/api/http.md

```diff
@@ -2027,6 +2027,7 @@ not abort the request or do anything besides add a `'timeout'` event.
 [`EventEmitter`]: events.html#events_class_eventemitter
 [`TypeError`]: errors.html#errors_class_typeerror
 [`URL`]: url.html#url_the_whatwg_url_api
+[`new URL()`]: url.html#url_constructor_new_url_input_base
```

Comment:
A nit: it seems this should go after `` [`net.createConnection()`]: `` ABC-wise.

Reply:
Whoops, fixed in the latest commit.